.github/fixtures/test-fixtures-locally.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR=$(readlink -f "$(dirname "$0")")
+
+if [ "$1" = "" ]; then
+    echo "Please input a fixture name."
+    exit 1
+fi
+
+export FIXTURES_DIR="$SCRIPT_DIR/$1"
+
+# Set up a temporary repository
+rm -rf /tmp/test
+mkdir -p /tmp/test
+cd /tmp/test
```

Comment:
This can be simplified to `cd $(mktemp -d)`

Reply:
e2a6a06f6db21b7eb4b6e7854f7ddb229b879178